src/components/LanguageSelection.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-menu location="end">
+    <template #activator="{ props }">
+      <v-list-item
+        :class="isInUserMenu ? `bg-surface-2 text-surface-1` : ``"
```

Comment:
I don't like the idea of modifying classes like this just to have some slight differences. I also don't have a better idea that works with vuetify.

In other projects I've used scoped classes where the parent would set a class on the component and either override in the parent or in component with the class the parent put on it.

@itsalaidbacklife, any idea of a better way to do this that's a little more vuetify friendly?

Reply:
@seriouslysean this seems like a good use case for the `variant` prop that would either be `"light"` or `"dark"` similar to the same prop that already exists in `TheUserMenu`, `TheHeader` and `TheFooter`. That would simply replacing the existing `isInUserMenu` prop with `variant` (which is expected to be 'light' or 'dark', perhaps using 'dark' as default.

As for the styles themselves I think classes applied within the component like this are reasonable. The parent sets the prop and the child can then consume the prop to do the styling. The main change then would be decoupling the prop name from the name `isInUserMenu` so that we can reuse the light and dark variants in other places e.g. the `GameView` at a future update. 

The only other thing I would change is refactoring this class calculation into a computed property so that the template code has less in it. That could work like:

```
const props = defineProps({
  variant: {
      type: string,
      default: 'dark',
    }
});
const activatorClass = computed(() => props.variant === 'dark' ? `bg-surface-2 text-surface-1` : '');
```
And in the template then we'd have

```
      <v-list-item
        :class="activatorClass"
```

Reply:
Thanks for the great feedback, guys! I'll apply these changes asap